templates/cisco_ios_show_ip_ospf_database_router.textfsm

```diff
@@ -0,0 +1,48 @@
+Value Filldown ROUTER_ID (\d+\.\d+\.\d+\.\d+)
+Value Filldown PROCESS_ID (\d+)
+Value Filldown AREA (\d+\.\d+\.\d+\.\d+|\d+)
+Value Filldown LSA_AGE (\d+)
+Value Filldown LSA_OPTIONS (.*)
```

Comment:
Can you change these from `.*` to `.+` since we're looking for one or more rather than 0 or more.

Reply:
Fixed via 22c9495